frontend/packages/console-shared/src/components/dropdown/index.ts

```diff
@@ -1 +1,2 @@
+export { default as DropdownWithSwitch } from './dropdown-with-switch/DropdownWithSwitch';
```

Comment:
This is a generalized version of https://github.com/openshift/console/blob/29377adf346a2c78c24bddd99b838726d7112a7b/frontend/packages/console-shared/src/components/namespace/NamespaceDropdown.tsx since PatternFly doesn't offer this functionality *and* `<NamespaceDropdown>` has functionality specific to `namespaces`/`projects`.